_data/other.yml

```
@@ -491,6 +491,15 @@ websites:
       twitter: spare5
       facebook: spare5app
       email_address: feedback@spare5.com
+      
+    - name: The Church of Jesus Christ of Latter-day Saints
+      url: https://www.lds.org/
+      img: lds.png
+      tfa: Yes
+      sms: Yes
+      phone: Yes
+      software: Yes
+      doc: https://account.lds.org/help/twoStepVerification # requires log in to view
```

While comments are technically valid in yaml, we've never used them before since the syntax is so simple and just to ensure compatibility with whatever might be ingesting our data.
```suggestion
      doc: https://account.lds.org/help/twoStepVerification
```

---

@davidschlachter please remove the comment.